x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/with_response_actions_role.ts

```diff
@@ -23,6 +23,8 @@ export const getWithResponseActionsRole: () => Omit<Role, 'name'> = () => {
             'execute_operations_all',
             'host_isolation_all',
             'process_operations_all',
+            'actions_log_management_all',
```

Comment:
I see you moved these around in the roles... Is it ok that `without_response_actions_role` (below) now has no access to actions log? (I'm assuming that is correct just based on the name of the role, but I did not implement the original, so just wanted to make sure you reviewed it with someone who knows this area better)

Reply:
I haven't really reviewed it with anyone before, but now seems to be a good time :) However all the other tests keep passing so my assumption is that it's fine, at least for the current state. If we would want to have some more specific tests then let's adjust. What do you think? 
@ashokaditya any thoughts?

Reply:
Here's how I see this. If you don't have response actions RBACs (isolate, release, other actions) but have read privilege to response actions history, then you should be able to see the actions log in endpoint details flyout as well as the response actions history page (with enterprise license). Even though you can't take response actions you should be able to view the actions history.

With this change, you won't be able to see history unless you also have response actions RBAC.